src/CSET/operators/plot.py

```diff
@@ -243,6 +243,24 @@ def _plot_and_save_contour_plot(
     # Add title.
     axes.set_title(title, fontsize=16)
 
+    # Add watermark with min/max/mean. Currently not user toggable.
+    axes.annotate(
+        "Min:"
+        + str(round(np.min(cube.data), 1))
+        + " Max:"
+        + str(round(np.max(cube.data), 1))
+        + " Mean:"
+        + str(round(np.mean(cube.data), 1)),
+        xy=(1, 0),
+        xycoords="axes fraction",
+        xytext=(-5, 5),
+        textcoords="offset points",
+        ha="right",
+        va="bottom",
+        size=11,
+        bbox=dict(boxstyle="round", fc="0.8", ec="0.5", alpha=0.9),
```

Comment:
```suggestion
        bbox=dict(boxstyle="round", fc=0.8, ec=0.5, alpha=0.9),
```
Keep numbers as numbers.

Reply:
OK, this seemed to break it. It looks like fc and ec are colours?

Reply:
OK, reading the code you had it right before. Maybe add a comment that the colour needs to be a string of a number to get a grey value?

Reply:
Sounds good - to be honest I leveraged some old code and never noticed the mix of string/float!

Reply:
In that case it might be clearer to use a hex colour code, as that is more widely used. That would be:
```py
fc = "#cccccc"
ec = "#808080"
```

Reply:
The comment covers this so don't worry about changing it.